i18n/en.json

```diff
@@ -2630,6 +2630,10 @@
     "id": "api.user.send_welcome_email_and_forget.failed.error",
     "translation": "Failed to send welcome email successfully"
   },
+  {
+    "id": "api.user.update_active.cannot_enable_guest_when_guest_feature_is_disabled.app_error",
+    "translation": "You cannot activate a guest account because guests feature is not enabled."
```

Comment:
Hi @jespino 
Thanks for making the change. Just a small correction in the error message to make it more consistent with the name of the feature. Can you please update it to:
`You cannot activate a guest account because Guest Access feature is not enabled.`

<img width="1279" alt="Screenshot 2019-11-15 at 7 00 41 PM" src="https://user-images.githubusercontent.com/1429138/68946960-3f086080-07da-11ea-88ff-63faa3ee8361.png">

Reply:
Fixed